zebra-state/src/service_traits.rs

```diff
@@ -0,0 +1,52 @@
+//! Trait aliases for Tower services used in state handling.
+//!
+//! These traits abstract over Tower `Service` implementations that
+//! handle state-modifying requests (`StateServiceTrait`) and
+//! read-only state requests (`ReadStateServiceTrait`).
+//!
+//! They enforce bounds such as `Clone`, `Send`, `Sync` and require the
+//! associated futures to be `Send`, making them suitable for
+//! multi-threaded async contexts.
+
+use crate::{BoxError, ReadRequest, ReadResponse, Request, Response};
+use tower::Service;
+
+/// Trait alias for services handling state-modifying requests.
+pub trait StateServiceTrait:
```

Comment:
I think it would be nice to name these `State` and `ReadState` traits, move these definitions to the `service` module (or a `zebra_state::service::traits` submodule), then only expose them through the `zebra_state::service` module so it's clear that it's a trait for types implementing state services.